src/screens/logged-out/login.tsx

```diff
@@ -24,16 +24,37 @@ const Login: React.FC = () => {
       alignItems="center"
       height="100vh"
       bgcolor="#f5f5f5"
+      sx={{
+        backgroundImage: "url('/src/static/img/login_background.jpg')", // Change to your image path
```

Comment:
I really don't know if this is the best way for a user to see the app for the first time. Personally I never really like that depressing black and white picture. Not sure if it's super attractive to new players.

I think a better approach would be to allow the user to see the home screen, e.g. " find games" screen or similar, and then click around and look at a few games.

We could have the login button somewhere on the screen. That way a user can get a sense of what the app is like before signing in.

Reply:
Actually this is still MVP. 

My plan is to make this (for web) a picture that is much more similar to the vDiplomacy - you get a header, see the login, but can scroll down to get a very easy explanation for the game. 

For app, that would have to be different - maybe a tutorial (swipe, swipe swipe) to go through that. For now, it's MVP and a bit more welcoming than the white empty background we have now :D. 

I would suggest we pull this - but it's temporary, and was temporary.